### ISSUE TITLE
Release of GLIP V2

### ISSUE THREAD

**Issue by AmrinKareem** [microsoft/GLIP#136]
Hi @hao-cheng @namisan @jwyang @csinva @ChunyuanLI - Thank you for your work. Could you please update when the code and models of GLIP V2 will be available? 
Thanks!

**Comment by dragen1860** [comment]
the  paper says code is already released at here.

**Comment by vatdut8994** [comment]
> the  paper(glipv2) says code is already released at here. 

I think they mean to say that it will be released here in the future. It states that the ArXiv paper has been released but I don't know whether they actually modified the code. Last modification was more than a year ago but the GLIPv2 news came out in late 2022. Really looking forward the the release!

**Comment by dragen1860** [comment]
> > the  paper(glipv2) says code is already released at here.
> 
> I think they mean to say that it will be released here in the future. It states that the ArXiv paper has been released but I don't know whether they actually modified the code. Last modification was more than a year ago but the GLIPv2 news came out in late 2022. Really looking forward the the release!

Yes, since the glipv2 code is not released yet, the paper should not say "is already". Thats confusing for an serious academic paper.

**Comment by Kingofolk** [comment]
3 months passed, has the code of glipV2 been released? There seems no update in this repo. Thx a lot.

**Comment by Vincent-luo** [comment]
@liunian-harold-li @Haotian-Zhang
Could you please update whether and when the code and models of GLIP V2 will be available? Thanks!
